runtime/src/lib.rs

```diff
@@ -553,7 +588,7 @@ impl_runtime_apis! {
 			add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
 			add_benchmark!(params, batches, pallet_balances, Balances);
 			add_benchmark!(params, batches, pallet_timestamp, Timestamp);
-			add_benchmark!(params, batches, encointer_communities, EncointerCommunities);
+			add_benchmark!(params, batches, pallet_template, TemplateModule);
 
```

Comment:
Was this intentional? If not, we probably also need to add the encointer_communities to the list_benchmarks above?

Reply:
@brenzi

Reply:
not intentional

Reply:
fixed in 672ed4e299935